src/ship/PlayerShipController.cpp

```diff
@@ -30,8 +30,9 @@ struct PlayerShipController::TotalDesiredAction {
 	vector3d linPower = vector3d(1.0);
 	// true means that desiredLinear contains speed, false means thrust
 	bool desireLinVel = false;
-	// true means that desiredAngular contains speed, false means thrust
```

Comment:
If not difficult, please correct the comment above `struct PlayerShipController::TotalDesiredAction`, about optional - that was the original idea.

Reply:
I'm not sure about this comment above stuct definition, do you think the two lines should be removed? : 
// vector3d (0.0) means we want zero speed, for example, so I made optional
// so that the system can say that it does not want to touch this parameter at all

Reply:
Yes, that will be enough